File: AN_2/SEM_2/MPP/c#/lab2/domain/Participant.cs
namespace MPP_Csharp.domain;

public class Participant
{
    public int IdParticipant { get; set; }
    public string Nume { get; set; }
    public string CNP { get; set; }
    public int CapacitateMotor { get; set; }
    public int? IdEchipa { get; set; } 

    public Participant(int id, string nume, string cnp, int capacitateMotor, int idEchipa)
    {
        IdParticipant = id;
        Nume = nume;
        CNP = cnp;
        CapacitateMotor = capacitateMotor;
        IdEchipa = idEchipa;
    }

    public override string ToString()
    {
        return $"{Nume} - {CapacitateMotor}cap_motor";
    }
}

File: AN_2/SEM_2/MPP/java/lab2/src/main/java/ro/mpp2024/domain/Participant.java
package ro.mpp2024.domain;

public class Participant {
    private int idParticipant;
    private String nume;
    private String CNP;
    private int capacitateMotor;
    private int idEchipa;

    public Participant(int idParticipant, String nume, String CNP, int capacitateMotor, int idEchipa) {
        this.idParticipant = idParticipant;
        this.nume = nume;
        this.CNP = CNP;
        this.capacitateMotor = capacitateMotor;
        this.idEchipa = idEchipa;
    }

    public int getIdParticipant() {
        return idParticipant;
    }

    public void setIdParticipant(int idParticipant) {
        this.idParticipant = idParticipant;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getCNP() {
        return CNP;
    }

    public void setCNP(String CNP) {
        this.CNP = CNP;
    }

    public int getCapacitateMotor() {
        return capacitateMotor;
    }

    public void setCapacitateMotor(int capacitateMotor) {
        this.capacitateMotor = capacitateMotor;
    }

    public int getIdEchipa() {
        return idEchipa;
    }

    public void setIdEchipa(int idEchipa) {
        this.idEchipa = idEchipa;
    }
}
